src/main/java/com/exadel/sandbox/controllers/AttachmentController.java

```diff
@@ -0,0 +1,57 @@
+package com.exadel.sandbox.controllers;
+
+import com.exadel.sandbox.dto.AttachmentDto;
+import com.exadel.sandbox.entities.Attachment;
+import com.exadel.sandbox.mapper.AttachmentMapper;
+import com.exadel.sandbox.service.AttachmentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("attachment")
+@RequiredArgsConstructor
+public class AttachmentController {
+    private AttachmentService attachmentService;
+    private AttachmentMapper attachmentMapper;
+
+    @GetMapping("list")
+    List<AttachmentDto> getAttachments(){
```

Comment:
Please apply code  formatting for all classes

Reply:
I fixed